src/main/webapp/script.js

```diff
@@ -54,20 +54,20 @@ function addObjects() {
   material = new THREE.LineBasicMaterial({color: 'red'});
   for (let increment = 0; increment < 15; increment++) {
```

Comment:
what does 15 represent?

Reply:
15 is just an arbitrary number to simulate what the actual pose trajectory should look like. I'll replace it with a variable to symbolize the number of points in database.